docs/topics/http/file-uploads.txt

```diff
@@ -122,6 +122,58 @@ field in the model::
             form = UploadFileForm()
         return render(request, 'upload.html', {'form': form})
 
+Uploading multiple files
+------------------------
+
+If you want to upload multiple files at the same time, you need to set the
+``multiple`` boolean attribute of the input element in your
+:class:`~django.forms.Form` first:
+
+.. snippet::
+    :filename: forms.py
+
+    from django import forms
+    from .models import ModelWithFileField
+
+    class FileFieldModelForm(forms.Form):
+        file_field = forms.FileField(widget=forms.ClearableFileInput(attrs={'multiple': True}))
+
+.. note::
+
+    Since we need to set the ``multiple`` boolean attribute of the input
+    element, we still need to define the ``file_field`` field even if we
+    are using :class:`~django.forms.ModelForm`.
+
+Then you can override the ``post`` method of your
+:class:`~django.views.generic.edit.FormView` subclass to handle multiple file
+uploads:
+
+.. snippet::
+    :filename: views.py
+
+    from django.views.generic.edit import FormView
+    from .forms import FileFieldModelForm
+    from .models import ModelWithFileField
+
+    # Imaginary function to handle an uploaded file.
+    from somewhere import handle_uploaded_file
+
+    class FileFieldView(FormView):
+        form_class = FileFieldModelForm
+        template_name = 'upload.html'
+        success_url = '/success/url/'
```

Comment:
Maybe it would make more sense to use `'...'` for the string values that the user needs to replace based on their project. (although in the case of `success_url`, it might be the result of `reverse()` too.